src/frequenz/sdk/microgrid/component/_component_data.py

```diff
@@ -130,17 +130,28 @@ class BatteryData(ComponentData):
     capacity: float
     """The capacity of the battery in Wh (Watt-hour)."""
 
-    power_lower_bound: float
+    power_inclusion_lower_bound: float
     """The maximum discharge power, in watts, represented in the passive sign
         convention. this will be a negative number, or zero if no discharging is
         possible.
     """
 
-    power_upper_bound: float
+    power_exclusion_lower_bound: float
+    """The minimum discharge power, in watts, represented in the passive sign
+        convention. this will be a negative number, or zero if no minimum discharge
+        power.
+    """
```

Comment:
I know you are following the same style that we had before, but ideally this should be like in methods, one short summary line, one empty line, and then a long optional description.

```py
    """The minimum discharge power, in watts, represented in the passive sign convention.

    This will be a negative number, or zero if no minimum discharge power.
    """
```

Same applies for the old `power_xxx_bound` docstrings.

Reply:
done